components/dashboard/src/teams/Members.tsx

```diff
@@ -179,9 +186,9 @@ export default function() {
                             <span className="flex-grow" />
                             <ItemFieldContextMenu menuEntries={m.userId === user?.id
                                 ? [{
-                                    title: 'Leave Team',
-                                    customFontStyle: 'text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300',
-                                    onClick: () => removeTeamMember(m.userId)
+                                    title: leaveTeamEnabled ? 'Leave Team' : 'Remaining owner',
+                                    customFontStyle: leaveTeamEnabled ? 'text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300' : 'text-gray-400 dark:text-gray-200',
```

Comment:
```suggestion
                                    customFontStyle: leaveTeamEnabled ? 'text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300' : 'text-gray-400 dark:text-gray-200 cursor-not-allowed',
```

How about also adapting the cursor to show that there is no action? E.g. `cursor-not-allowed` or `cursor-default`.

Reply:
Given that the label is changed to `Remaining owner`, I don't think it's necessary to change the cursor. Also, there is no other example in the dashboard to provide feedback using the "not-allowed" cursor.

Reply:
Would it make sense to have some server-side validation for this as well?